_docs/platforms/kubernetes/install/1.12.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: guide
+title: StorageOS Docs - Installing on Kubernetes 1.12
+anchor: platforms
+module: platforms/kubernetes/install/1.12
+redirect_from: /docs/install/schedulers/kubernetes
+redirect_from: /docs/install/kubernetes
+---
+
+# Kubernetes 1.12+
+
+```bash
+# Install StorageOS as a daemonset with RBAC support
+git clone https://github.com/storageos/deploy.git storageos
+cd storageos/k8s/deploy-storageos/standard
+./deploy-storageos.sh
+```
+
+or using the Helm chart:
+```bash
+helm repo add storageos https://charts.storageos.com
+helm repo update
+# Set cluster.join to hostnames or ip addresses of at least one node
+helm install storageos/storageos --name my-release --namespace storageos --set cluster.join=node01,node02,node03
+
+# Follow the instructions printed by helm install to update the link between Kubernetes and StorageOS. They look like:
+$ ClusterIP=$(kubectl get svc/storageos --namespace storageos -o custom-columns=IP:spec.clusterIP --no-headers=true)
+$ ApiAddress=$(echo -n "tcp://$ClusterIP:5705" | base64)
+$ kubectl patch secret/storageos-api --namespace storageos --patch "{\"data\":{\"apiAddress\": \"$ApiAddress\"}}"
+```
+
+## Installation with CSI (Container Storage Interface)
+
+CSI allows you to set StorageOS features (`storageos.com/*` labels) in
+the StorageClass, but not PVCs. If you need to set labels in PVCs (but not
+StorageClasses) or your environment does not support CSI, you may install
+StorageOS without CSI:
```

Comment:
I'm not sure if we should mention this here. Maybe in some troubleshooting page, or maybe below the following commands.

Reply:
We will add  post GA.